src/helm/config/tokenizer_configs.yaml

```diff
@@ -128,6 +128,11 @@ tokenizer_configs:
       class_name: "helm.proxy.tokenizers.vertexai_tokenizer.VertexAITokenizer"
     end_of_text_token: "</s>"
     prefix_token: ""
+  - name: google/text-bison@002
+    tokenizer_spec:
+      class_name: "helm.proxy.tokenizers.vertexai_tokenizer.VertexAITokenizer"
+    end_of_text_token: "</s>"
+    prefix_token: ""
```

Comment:
Why do we have a new tokenizer here? It seems to be exactly the same as `google/text-bison@001`. Do they provide different tokenizers in `vertexai` if we provide a different name? If so maybe elt's add a comment here because for someone who forgot that the tokenizer name might be passed to the `TokenizerClient`, it just looks like we duplicated a tokenizer.

Reply:
The tokenizer name gets passed to the Tokenizer class. This will be documented in ReadTheDocs so there is no need to add a comment here specifically.